java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -315,6 +315,16 @@ public OnnxModelMetadata getMetadata() throws OrtException {
     return metadata;
   }
 
+  /**
+   * Returns the timestamp that profiling started in nanoseconds.
+   *
+   * @return the profiling start time in ns.
+   * @throws OrtException If the native call failed.
+   */
+  public long getProfilingStartTime() throws OrtException {
```

Comment:
Should we call this getProfilingStartTimeInNs (taking cue from https://docs.oracle.com/javase/7/docs/api/java/util/Calendar.html#getTimeInMillis())?

Reply:
Sure.